content/en/blog/_posts/2021-12-15-pod-security-admission-beta.md

```diff
@@ -0,0 +1,774 @@
+
+## How Does Pod Security Work?
+    
+Pod Security is a built-in [admission controller](/docs/reference/access-authn-authz/admission-controllers/) starting with Kubernetes v1.22, but can also be run as a standalone [webhook](/docs/reference/access-authn-authz/extensible-admission-controllers/). Admission controllers function by intercepting requests in the Kubernetes API server prior to persistence to storage. They can either `admit` or `deny` a request. In the case of Pod Security, pod specifications will be evaluated against a configured policy in the form of a Pod Security Standard. This means that security sensitive fields in a pod specification will only be allowed to have [specific](h/docs/concepts/security/pod-security-standards/#profile-details) values. 
+
+## Configuring Pod Security
+    
+### Pod Security Standards
+    
+In order to use Pod Security we first need to understand [Pod Security Standards](/docs/concepts/security/pod-security-standards/). These standards define three different policy levels that range from permissive to restrictive. These levels are as follows:
+   * Privileged — open and unrestricted
+   * Baseline — Covers known privilege escalations while minimizing restrictions
+   * Restricted — Highly restricted, hardening against known and unknown privilege escalations. May cause compatibility issues
+
+Each of these policy levels define which fields are restricted within a pod specification and the allowed values. Some of the fields restricted by these policies include:
+   * `spec.securityContext`
+   * `spec.hostNetwork`
+   * `spec.volumes[*].hostPath`
+   * `spec.containers[*].securityContext`
+
+Policy levels are applied via labels on Namespace resources, which allows for granular per-namespace policy selection. The AdmissionConfiguration in the api server can also be configured to set cluster-wide default levels and exemptions.
+
+### Policy modes
+
+Policies are applied in a specific mode. Multiple modes (with different policies) can be set on the same namespace. Here is a list of modes:
+   * `enforce` — Any Pods that violate the policy will be rejected
+   * `audit` — Violations will be recorded as an annotation in the audit logs, but don't affect whether the pod is allowed.
+   * `warn` — Violations will send a warning message back to the user, but don't affect whether the pod is allowed.
+
+In addition to modes you can also pin the policy to a specific version for example v1.22. Pinning to a specific version allows the behavior to remain consistent as the policy definition changes over Kubernetes releases. If pinning to a specific Pod Security Standard version in a Kubernetes release you should take the time to understand the Kubernetes [supported version policy](/releases/version-skew-policy/#supported-versions).
```

Comment:
`... (for example, v1.22).`

Reply:
`Pinning to a specific version allows the behavior to remain consistent if the policy definition changes in future Kubernetes releases.`

Reply:
> ... you should take the time to understand the Kubernetes [supported version policy](/releases/version-skew-policy/#supported-versions).

I'm not sure that's particularly relevant... PodSecurity will continue to honor specific versions of the policy indefinitely.

Reply:
Thanks Jordan - the intent of this statement was if there was a specific bug in a Pod Security Standard that needed fixing and you were using a pinned version you may be out of luck depending on support. Not sure if that would help anyway because I'm not sure you can pin to a patch release (eg. v1.22.4)

Reply:
Yanking that sentence